tests/test_install.ts

```diff
@@ -187,6 +188,34 @@ describe("Agama test", function () {
 
     // For development will be useful to stop before starting installation
     if (agamaInstall === true) {
+        it("should create first user", async function (){
+            await page.locator("a[href='#/users']").click();
+
+            let button: any = await Promise.any([
+                page.waitForSelector("a[href='#/users/first']"),
+                page.waitForSelector("button#actions-for-" + agamaUser)
```

Comment:
no worries to hardcode this, in openQA is a long hardcoded user, honoring the creator.
In more advance test we with think about a better selector.

Reply:
Hardcoding never is a good idea...

Reply:
bernand user hasn't change in +10 years, no worries, it is a constant.